pkg/filterengine/with_all_filters.go

```diff
@@ -15,11 +15,10 @@ const (
 )
 
 // WithAllFilters returns new DefaultFilterEngine instance with all filters registered.
-func WithAllFilters(logger *logrus.Logger, dynamicCli dynamic.Interface, mapper meta.RESTMapper, res []config.Resource) *DefaultFilterEngine {
+func WithAllFilters(logger *logrus.Logger, dynamicCli dynamic.Interface, mapper meta.RESTMapper, _ []config.Resource) *DefaultFilterEngine {
```

Comment:
If we don't need `res []config.Resource`, don't pass it 🙂 

```suggestion
func WithAllFilters(logger *logrus.Logger, dynamicCli dynamic.Interface, mapper meta.RESTMapper) *DefaultFilterEngine {
```